### PR TITLE
Minor expand/correct MacOS build instructions

### PR DESCRIPTION
**Pull request by Btijmen**
When preparing to build locally, I had to install wasm_pack as well or yarn wouldn't work. Hence, I added install instructions for wasm_pack

### CODE REVIEW

File: BUILD_macos.md [hunk 1]
```diff
@@ -81,6 +81,11 @@ PRs with corrections and updates are welcome!
   * `brew install node yarn`
   * Currently, [`yarn`](https://classic.yarnpkg.com/en/docs/install) (< 2.0.0) is required
 
+* Yarn uses wasm-pack, so if that's not present, we need to install it
+  ```shell
+  curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+  ```
+
```
Comment (zmb3) [inline BUILD_macos.md:88]:
```suggestion
* Teleport uses [wasm-pack](https://rustwasm.github.io/wasm-pack/),
  so if that's not present, we need to [install](https://rustwasm.github.io/wasm-pack/installer/) it
```

- Yarn doesn't use wasm-pack, Teleport does
- There are several ways to install it, so let's just link to the installation docs.

Comment (Btijmen) [reply BUILD_macos.md:88]: Fair enough, thanks for the input

File: BUILD_macos.md [hunk 2]
```diff
@@ -81,6 +81,9 @@ PRs with corrections and updates are welcome!
   * `brew install node yarn`
   * Currently, [`yarn`](https://classic.yarnpkg.com/en/docs/install) (< 2.0.0) is required
 
+* Teleport uses wasm-pack, so if that's not present, we need to install it. Find the wasm install docs [here](https://rustwasm.github.io/wasm-pack/installer/).
+
+
```
Comment (zmb3) [inline BUILD_macos.md:86]:
```suggestion
```

Remove extra newline.

File: BUILD_macos.md [hunk 3]
```diff
@@ -81,6 +81,9 @@ PRs with corrections and updates are welcome!
   * `brew install node yarn`
   * Currently, [`yarn`](https://classic.yarnpkg.com/en/docs/install) (< 2.0.0) is required
 
+* Teleport uses wasm-pack, so if that's not present, we need to install it. Find the wasm install docs [here](https://rustwasm.github.io/wasm-pack/installer/).
```
Comment (zmb3) [inline BUILD_macos.md:84]:
```suggestion
* Teleport uses wasm-pack, so if that's not present, we need to [install it]( https://rustwasm.github.io/wasm-pack/installer/).
```

WASM and wasm-pack are not the same thing.

Comment (Btijmen) [reply BUILD_macos.md]: Correct, fixed

File: BUILD_macos.md (hunk 2 above)
Comment (Btijmen) [reply BUILD_macos.md:85]: Done!

**Review by zmb3 — APPROVED** [review]

**Review by rosstimothy — APPROVED** [review]

**Comment by zmb3** [comment]
Looks like wasm-pack is already present in this file.

**Pull request closed without merging**